### DISCUSSION TITLE
Which PHP Libraries are included in the php install in brew and how do I install other libraries?

### DISCUSSION THREAD

**Discussion by TheOnePossum** [Homebrew/discussions#5580] [Getting started]
### Output of `brew config`

```shell
Waiting for the Server, Haven't been able to test yet
```

### Output of `brew doctor`

```shell
Waiting for the Server, Haven't been able to test yet
```

### Description of issue

I want to use Brew to turn a Mac mini into a Webserver to host vBulletin and Mediawiki. These two require a bunch of PHP libraries, to be specific:

- mysqli_nd
- mysqlnd
- PHAR 
- GD
- Hash
- JSON
- XML
- mbstring
- iconv
- cURL
- xmlreader
- dom
- intl
- apcu

How can I check wether these are installed with php https://formulae.brew.sh/formula/php#default and add them if they aren't ?

**Comment by SMillerDev** [comment]
You can run `php -m` after installation to get a list of modules

**Reply by TheOnePossum** [reply]
Thanks!

**Comment by TheOnePossum** [comment]
For anyone having the same question: The default php installation includes all modules but apcu. Apcu however is only recommended with Mediawiki so there is that.

Edit:
It can apparently be installed with 
`pecl install apcu`
which might throw the error
`pcre2.h not found`
which might be possible to fix with https://stackoverflow.com/questions/72039019/pcre2-h-no-such-file-or-directory
